src/scripts.js

```diff
@@ -21,6 +21,8 @@ const selectedRecipeView = document.querySelector('.individual-recipe-container-
 const filterBar = document.querySelector('.filter-section-js');
 const mainView = document.querySelector('.main-view-container-js');
 const searchRecipesButton = document.getElementById('searchRecipes');
+const filterButton = document.getElementById('filterButton');
```

Comment:
yes, from what I remember this also seems like the right direction! I think we want to select ALL checkboxes and see which ones are checked (aka have a value of true). I'm wondering if we need to find the id equivalent of querySelectAll thought. I'm not sure if that exists but it seems like we are only selecting one element right here rather than the whole group.